twentytwentyone-blocks/block-template-parts/footer.html

```diff
@@ -0,0 +1,19 @@
+<!-- wp:spacer {"height":70} -->
+<div style="height:70px" aria-hidden="true" class="wp-block-spacer"></div>
```

Comment:
Would you say this should be better expressed as top padding on the `footer` template part if we had such a control?

Reply:
That would certainly be one way to do it — it would come in handy for the header too. At the moment, those template parts are truly just empty wrappers with no settings (other than alignment, I guess). But I could see them being a little more full-featured: padding, background color, etc.

Reply:
An additional, traditional .site-footer class name with styles works too ;)

Reply:
Template parts (at least these two) should be more "robust" — output a semantic tag, have style attributes, and yes, a class :)

It might be nice to conceptualize template parts as `site-` fragments. So by default template parts would output a class of `site-{template-part-name}`.